nvtabular/graph/graph.py

```diff
@@ -55,8 +79,6 @@ def fit_schema(self, root_schema: Schema) -> "Graph":
         )
         self.output_schema = self.output_node.output_schema
 
-        return self
```

Comment:
I think this was removed accidentally, but `fit_schema` should probably still `return self`

Reply:
fit schema still has return self. Its just up further in the code.